app/services/katello/pulp3/content_view_version/import.rb

```diff
@@ -121,11 +121,20 @@ def self.find_or_create_import_view(organization:, metadata:, library: false)
                         }
           end
 
+          fail _("Content View label not provided.") if metadata[:label].blank?
+
           cv = ::Katello::ContentView.find_by(label: metadata[:label],
-                                              organization: organization,
-                                              import_only: true)
+                                              organization: organization)
           if cv.blank?
             ::Katello::ContentView.create!(metadata.merge(organization: organization, import_only: true))
+          elsif !cv.import_only?
+            msg = _("Unable to import in to Content View specified in the metadata - '%{name}'. "\
+                     "The 'import_only' attribute for the content view is set to false. "\
+                     "To mark this Content View as importable, have your system administrator"\
+                     " run the following command on the satellite. "\
```

Comment:
Nice! Recommend removing the 'on the satellite' part so it works better up and down-stream

Reply:
Does this work ??


```suggestion
                     " run the following command on the main server. "\
```